### PR TITLE
padding and divider changes

### PR DESCRIPTION
**Pull request by DavidNic11**

### CODE REVIEW

File: public/react/Configure/components/Teams/ConfigureTeamsForm.tsx [hunk 1]
```diff
@@ -40,86 +42,91 @@ const ConfigureTeamsForm: FC<ConfigureTeamsFormProps> = ({
...
+          onSave={update}
+        />
+        <InheritanceToggleField
+          isInheriting={!userData.spreadEffortAcrossDates && typeof userData.spreadEffortAcrossDates === "object"}
+          onInheritanceChange={(shouldCustomize) => toggleInheritance("spreadEffortAcrossDates", shouldCustomize)}
+          name="spreadEffortAcrossDates"
+          control={control}
+          onSave={update}
+          label="Spread effort"
+          description="Spread estimate accross dates"
```
Comment (bmomberger-bitovi) [inline public/react/Configure/components/Teams/ConfigureTeamsForm.tsx:78]: oh ffs.  "across"  -- one C.

**Review by bmomberger-bitovi — CHANGES_REQUESTED** [review]
One change.  Everything else looks good.

File: public/react/Configure/components/Teams/AllTeamsDefaultsForm.tsx [hunk 2]
```diff
@@ -96,7 +101,7 @@ const AllTeamsDefaultForm: FC<AllTeamsDefaultFormProps> = ({ save, userData, aug
           control={control}
           onSave={update}
           label="Spread effort"
-          description="Spread estimate access dates"
+          description="Spread estimate accross dates"
```
Comment (bmomberger-bitovi) [inline public/react/Configure/components/Teams/AllTeamsDefaultsForm.tsx:104]: here too.

File: public/react/Configure/components/Teams/ConfigureTeamsForm.tsx (hunk 1 above)
Comment (DavidNic11) [reply public/react/Configure/components/Teams/ConfigureTeamsForm.tsx:78]: that's like, your opinion man

**Review by bmomberger-bitovi — APPROVED** [review]
:shipit:

**Pull request merged**